config/locales/en.yml

```diff
@@ -254,6 +254,11 @@ en:
           short: "Short name"
       notice:
         remember_items_and_projects: "Remember to set items and projects in the respective tabs for this custom field."
+      hierarchy:
+        subitems:
+          zero: no sub-items
```

Comment:
🟡 "zero" is currently a dead locale key, as those are not rendered. I'd rather remove it and add it, once we need this text.

Reply:
This is just protection against weird cases. :P